spec/hdinsight_application_service_spec.rb

```diff
@@ -0,0 +1,60 @@
+#####################################################################################
+# hdinsight_application_service_spec.rb
+#
+# Test suite for the Azure::Armrest::HDInsight::HDInsightApplicationService class.
+#####################################################################################
+require 'spec_helper'
+
+describe "HDInsight::HDInsightApplicationService" do
+  before { setup_params }
+  let(:sds) { Azure::Armrest::HDInsight::HDInsightApplicationService.new(@conf) }
+
+  context "inheritance" do
+    it "is a subclass of ResourceGroupBasedSubservice" do
+      ancestors = Azure::Armrest::HDInsight::HDInsightApplicationService.ancestors
+      expect(ancestors).to include(Azure::Armrest::ResourceGroupBasedSubservice)
+    end
+  end
+
+  context "constructor" do
+    it "returns a Sql::SqlDatabaseService instance as expected" do
```

Comment:
A typo here. Evidently you copied and pasted an existing spec. Can you change the name `sds` to something more meaning to this test, such as `hd_ias`?